pinecone-mixin/config.libsonnet

```diff
@@ -0,0 +1,31 @@
+{
+  _config+:: {
+    local this = self,
+    dashboardTags: ['pinecone'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+    dashboardNamePrefix: '',
+    uid: 'pinecone',
+    
+    // Filtering and labels
+    filteringSelector: 'job=~"prometheus.scrape.pinecone_metrics"', 
+    groupLabels: [],  // Options: ['cloud'], ['region'], ['cloud', 'region'], or [] for single cloud/region
+    instanceLabels: ['index_name'],  // Each index is an instance
```

Comment:
Could you elaborate here on what an instance represents? Or maybe not here, but in the readme?